--- /dev/fd/63	2026-09-17 13:45:41.192326106 -0400
+++ /dev/fd/62	2026-09-17 13:45:41.192326106 -0400
@@ -1,5 +1,5 @@
 {
-  "candidateFingerprint": "b04e366b120bf287de982f1b16d448ce726e6ed2ef907606f1e4febd9d282cb1",
+  "candidateFingerprint": "8f984d0042f42907a797a52d956ef9051fe62952c6c43dbc402eef7193416fea",
   "candidateId": "FIFTH_SPACE_CENSUS_v0",
   "companionChecks": {
     "governsOutDegree": {
@@ -581,7 +581,7 @@
   "evidenceBindings": {
     "canonicalLedgerSha256": "e6570972260fdae5c4ca878272dc89a9ff353d48762eefbe019707d229cd242d",
     "courtRootedPositionsSha256": "85100e4bb5185b2630d1607a35e417fc90fdd5bb4ae63f0e775224d6b1436fec",
-    "decisionLedgerSha256": "baa5db55de37c72e6fe5c7e2430f82d2ed04c9ff50914ffee8ba67bf412f14da",
+    "decisionLedgerSha256": "ac50241b6c432394b55864e2ea79755d70ddca60529476edf470f2c39f509c74",
     "networkFingerprint": "21e2a632837ecf40fe9229e9eb4ec0a5cceb9e2043fe89cb8e1d320518d7bdbc",
     "observationLedgerSha256": "20793bdc3c509a84951a39578a40285ed8205f87033a12f494000fe712e8da52"
   },
